default.nix

```diff
@@ -0,0 +1,16 @@
+let
+  base-nixpkgs = import <nixpkgs> {};
+  mozillaOverlay = import (builtins.fetchTarball https://github.com/mozilla/nixpkgs-mozilla/archive/master.tar.gz);
```

Comment:
Recommend pinning `base-nixpkgs` and `mozillaOverlay` using `niv` or `fetchTarball`.

Reply:
@davidrusu please see my PR on your `support-v5-and-svg-output` branch.